ZFSin/spl/module/spl/spl-kmem.c

```diff
@@ -63,9 +63,9 @@
 // proxy to the machine experiencing memory pressure.
 //
 // xnu vm variables
-extern volatile unsigned int vm_page_free_wanted; // 0 by default smd
-extern unsigned int vm_page_free_min; // 3500 by default smd kern.vm_page_free_min, rarely changes
-extern volatile unsigned int vm_page_free_count; // will tend to vm_page_free_min smd
+extern volatile uint64_t vm_page_free_wanted; // 0 by default smd
```

Comment:
it seems that unsigned int for this variable will be ok. it's only when the computation is made with * that the size can go beyond unsigned int but in such case the result is saved in a different uint64 variable already.  
General comment: some files are shared with linux and Jorgen will update them once in a while so if a file is one of those then we can't change the type without understanding deeper.  did you get a compile warning for that one and the ones related below ?

Reply:
Yes, all data type changes in this PR are made to address compiler warnings.